modules/campaignion_petition/campaignion_petition_templates/campaignion_petition_templates.features.uuid_node.inc

```diff
@@ -10,6 +10,61 @@
  */
 function campaignion_petition_templates_uuid_features_default_content() {
   $nodes = array();
+
+  $nodes[] = array(
+  'title' => 'Thank you! You’re awesome!',
+  'log' => '',
+  'status' => 0,
+  'comment' => 0,
+  'promote' => 0,
+  'sticky' => 0,
+  'type' => 'thank_you_page',
+  'language' => 'en',
+  'created' => 1491564270,
+  'tnid' => 0,
+  'translate' => 0,
+  'uuid' => '0e6c7ce6-5701-4c17-b306-be7eba55dfe8',
+  'body' => array(),
+  'campaignion_overlay_options' => array(),
+  'field_main_image' => array(),
+  'field_reference_to_campaign' => array(),
+  'share_light' => array(
+    'und' => array(
+      0 => array(
+        'toggle' => 1,
+        'options' => array(
+          'subject' => 'Share this action!',
+          'image' => array(
+            'fid' => 0,
+          ),
+          'counter_toggle' => 1,
+          'channels' => array(
+            'email' => array(
+              'toggle' => TRUE,
+            ),
+            'facebook' => array(
+              'toggle' => TRUE,
+            ),
+            'twitter' => array(
+              'toggle' => TRUE,
+              'text' => 'Tweet tweet for kittens!',
+            ),
+          ),
+          'link' => array(
+            'path' => '',
+          ),
+        ),
+      ),
+    ),
+  ),
+  'path' => array(
+    'pathauto' => FALSE,
+    'alias' => 'content/thank-you-you’re-awesome',
+  ),
+  'date' => '2017-04-07 11:24:30 +0000',
+  'user_uuid' => '9dbdd05e-9e99-4f34-a31b-8685d3f6bcc3',
+  'webform' => NULL,
+);
```

Comment:
I thought you wanted to remove the exported thank you pages? ;)

Reply:
I didn’t suggest to do it in thie PR though.